test/Benchmarks/src/Vector/Array_Proxy_Bench.enso

```diff
@@ -32,7 +32,7 @@ type Data
 
 
 collect_benches = Bench.build builder->
-    vector_size = 10000
+    vector_size = 1000 * 1000
```

Comment:
What was dictating these changes?

Reply:
From the PR description:
```
Tweak some warmup / measurement configurations for some benchmark such that they have enough time for warmup.
```

Optimally, we want to have roughly 5 seconds for warm up and 5 seconds for the measurement phase for each benchmark because we want the whole job to run no longer than 90 minutes and at the same time, we would like to have a reasonable number of benchmarks. From the perspective of performance measurement, 5 seconds for *warmup* is far too low. But it is sufficient for our purposes - we do not strive to publish a paper about the Engine performance after all.

I was driven by a rule of thumb: The score of each benchmark should be roughly between 0.5 ms/op and 500 ms/op. That way, it is ensured that the benchmark is roughly invoked at least 20 times in the warmup phase.

In this particular case, benchmarks in `Array_Proxy_Bench` were too fast - the score was lower than 0.005 ms/op, so I increased the data size.